pages/common/sed.md

```diff
@@ -2,21 +2,21 @@
 
 > Run replacements based on regular expressions
 
-- replace all occurrences of a string in a file, and print the result
+- replace the first occurrence of a string in a file, and print the result
 
-`sed 's/{{find}}/{{replace}}/g' {{filename}}`
+`sed 's/{{find}}/{{replace}}/' {{filename}}`
 
-- replace all occurrences of a string in a file, and overwrite the file
+- replace all occurrences of a string in a file, overwriting the file
   contents
 
-`sed -i '' 's/{{find}}/{{replace}}/g' {{filename}}`
+`sed --in-place 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
Do you think being more verbose is more useful? Because as a rule, we don't follow that rule. As in, we're not consistent but we usually don't even if it would make it easier to remember.